civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc

```diff
@@ -2054,7 +2054,7 @@ return date(\'Y-m-d\');';
   $handler->display->display_options['header']['area_text_custom']['table'] = 'views';
   $handler->display->display_options['header']['area_text_custom']['field'] = 'area_text_custom';
   $handler->display->display_options['header']['area_text_custom']['empty'] = TRUE;
-  $handler->display->display_options['header']['area_text_custom']['content'] = '<a href="/node/add/hr-documents" class="btn btn-primary btn-lg"><i class="fa fa-plus-circle"></i>&nbsp;Add HR Document</a>';
```

Comment:
I think links must be managed with l function, and if you are going to send text to this function you should be using t function. Something like "l(t("text"),'path')". You should use some l argument options to achieve it.